app/sessions.js

```diff
@@ -52,36 +52,21 @@ class UserSession extends Session {
      */
     async setAccounts() {
         anchor.setProvider(this.provider);
+        const uuidBytes = this.uuid.toArrayLike(Buffer, 'le', 8);
         [this.reqAccount, this.reqBump] = await anchor.web3.PublicKey.findProgramAddress(
-            [Buffer.from(this.seed), this.keypair.publicKey.toBuffer()],
+            [Buffer.from(this.seed), this.keypair.publicKey.toBuffer(), uuidBytes],
             this.programId
             );
-
-        [this.vaultAccount, this.vaultBump] = await anchor.web3.PublicKey.findProgramAddress(
-            [Buffer.from(this.vaultSeed), this.keypair.publicKey.toBuffer()],
-            this.programId
-            );
-    }
-
-    // Only use this locally
-    async airdropVaultAccount(amount=1000000000) {
-        anchor.setProvider(this.provider);
-
-        await this.provider.connection.confirmTransaction(
-            await this.provider.connection.requestAirdrop(this.vaultAccount, amount),
-            "confirmed"
-        );
     }
 
     async initializeAccount() {
         anchor.setProvider(this.provider);
         await this.program.rpc.initialize(
```

Comment:
Minor, up to you to fix in this PR or not. `this.program.rpc` syntax is being deprecated in favor of `this.program.methods`

In the new syntax, I think this would be:
```suggestion
        await this.program.methods.initialize(
            this.reqBump,
            this.uuid)
        .accounts(
            {
                                    requester: this.reqAccount,
                    vault: this.vaultAccount,
                    authority: this.keypair.publicKey,
                    oracle: this.oraclePubkey,
                    rent: anchor.web3.SYSVAR_RENT_PUBKEY,
                    systemProgram: anchor.web3.SystemProgram.programId,
            }
        )
        .signers([this.keypair])
        .rpc()
```